src/landing/home.css

```diff
@@ -9,6 +9,33 @@
   grid-template-rows: 1fr auto 1fr;
 }
 
+.donate-home {
+  display: block;
+  position: relative;
+  background-image: url("/assets/CS_bg_img_2.png");
+  background-size: 744px;
+  background-repeat: repeat-x;
+  color: white;
+  text-decoration: blink;
+  font-weight: 500;
+  font-size: 20px;
+  border-radius: 20px;
+  padding: 5px 10px;
+  animation: scroll-it 360s linear infinite;
+  margin-top: 10px;
+  z-index: 100;
+  width: 99px;
+  left: 11px;
+  height: 43px;
+  text-align: center;
+}

```

Comment:
don't repeat all of this and the animations 3 times across the project. just set up the common styles once in a single place and add additional positioning rules as needed